frontend/apps/marketing/src/contentful/register-custom-components.ts

```diff
@@ -3,14 +3,11 @@
  *
  * Note: This file must be imported both server-side and client-side to ensure Contentful is able to map on both rendering modes.
  */
+import Divider, {DividerContentfulComponentDefinition} from '@/components/divider';
```

Comment:
Any reason why this got moved ahead of the Contentful imports below?

Reply:
Oh I guess the IDE put it there. We don't have the import order rules yet (this is being discussed by the standards group). I think we'll have this fixed once the import order linter rule comes in!

Reply:
Cool, ty!